package.json

```diff
@@ -23,21 +23,33 @@
   "workspaces": [
     "easy-ui-icons",
     "easy-ui-react",
-    "easy-ui-storybook",
     "easy-ui-tokens",
     "easy-ui-tsconfig",
     "stylelint-easy-ui",
     "eslint-config-easy-ui"
   ],
   "scripts": {
     "clean": "turbo run clean && rm -rf ./node_modules",
-    "format": "prettier . --write"
+    "format": "prettier . --write",
+    "storybook": "start-storybook -p 9003",
+    "build-storybook": "build-storybook"
   },
   "devDependencies": {
+    "@babel/core": "^7.20.12",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-essentials": "^6.5.16",
+    "@storybook/addon-interactions": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
+    "@storybook/builder-vite": "^0.3.0",
+    "@storybook/react": "^6.5.16",
+    "@storybook/testing-library": "^0.0.13",
+    "@vitejs/plugin-react": "^3.1.0",
+    "babel-loader": "^8.3.0",
```

Comment:
unfortunate that we have this bloat in the root package but it seems to be the best place it makes sense for a design system with multiple packages to have potential stories in each